downstream/modules/eda/con-custom-credential-types.adoc

```diff
@@ -0,0 +1,62 @@
+
+* Environment variables
+* Ansible extra variables
+* File-based templating, which means generating .ini or .conf files that contain credential values
+
+//[J.Self] Reviewers: please confirm if this is true. I borrowed it from automation controller content, per suggestions from several SMEs and one of our principal tech writers. I just want to confirm technical accuracy here, especially the last line as we might not have "machine credentials".
+You can attach one SSH and multiple cloud credentials to a job template. Each cloud credential must be of a different type. Only one of each type of credential is permitted. Vault credentials and machine credentials are separate entities.
+
+[NOTE]
+====
+* When creating a new credential type, you must avoid collisions in the `extra_vars`, `env``, and file namespaces.
+* Environment variable or extra variable names must not start with *EDA_* because they are reserved.
+* You must have System administrator (superuser) permissions to be able to create and edit a credential type and to be able to view the *Injector configuration* field.
+====
+
+When you customize your own credential types, they will display on the Credential Types page along with a list of built-in credential types.
+
+Each credential type displays its own unique configurations in the Input Configuration and the Injector Configuration fields, if applicable. Both YAML and JSON formats are supported in the configuration fields.
+//Note from J. Self: REVIEWERS, please confirm the Note above along with the paragraph about attachning one SSH and multiple clouds to a job template. I copied this from automation controller content, but not entirely sure it's relevant to EDA.
+
+[discrete]
+== Input Configuration
+
+The Input configuration has two attributes:
+
+* fields - a collection of properties for a credential type.
+* required - a list of required fields.
+
+Fields can have multiple properties, depending on the credential type you select.
+
+.Input Configuration Field Properties
+[cols="a,a,a"]
+|===
+| Fields | Description | Mandatory (Y/N)
+
+h| id | Unique id of the field; must be a string type and stores the variable name | Yes
+
+h| type | Can be string or boolean type | No, default is string
+
+h| label | Used by the UI when rendering the UI element | Yes
+
+h| secret | Will be encrypted | No, default false
+
+h| multiline | If the field contains data from a file the multiline can be set to True | No, default false
+
+h| help_text | The help text associated with this field | No
+
+|===
+
+[discrete]
+== Injector Configuration
+
+You can use the Injector configuration field to take the fields from input configuration field and map them into `extra_vars` that can be sent to ansible-rulebook when running the activation. The Injector currently only supports `extra_vars`. 
```

Comment:
@jself-sudoku "The Injector currently only supports `extra_vars`. " This is correct so earlier references to env vars and file is incorrect.